Src/WitsmlExplorer.Api/Workers/Delete/DeleteEmptyMnemonicsWorker.cs

```diff
@@ -0,0 +1,166 @@
+        private readonly IMnemonicService _mnemonicService;
+
+        public DeleteEmptyMnemonicsWorker(
+            ILogger<DeleteEmptyMnemonicsJob> logger,
+            IWitsmlClientProvider witsmlClientProvider,
+            IWellboreService wellboreService,
+            ILogObjectService logObjectService,
+            IMnemonicService mnemonicService)
+            : base(witsmlClientProvider, logger)
+        {
+            _wellboreService = wellboreService;
+            _logObjectService = logObjectService;
+            _mnemonicService = mnemonicService;
+        }
+
+        public override async Task<(WorkerResult WorkerResult, RefreshAction RefreshAction)> Execute(DeleteEmptyMnemonicsJob job)
+        {
+            IWitsmlClient client = GetTargetWitsmlClientOrThrow();
+
+            var wellboreRefsToCheck = job.Wellbores.ToList();
+
+            wellboreRefsToCheck.AddRange(await ExtractWellboreRefs(job.Wells));
+
+            var reportItems = new List<DeleteEmptyMnemonicsReportItem>();
+
+            foreach (var wellboreRef in wellboreRefsToCheck)
+            {
+                var logsToCheck = await ExtractLogs(wellboreRef);
+
+                if (!logsToCheck.IsNullOrEmpty())
+                {
+                    foreach (var logToCheck in logsToCheck)
+                    {
+                        var logCurves = await GetLogCurveInfos(logToCheck);
+
+                        var mnemonicsToDelete = FindNullMnemonics(job.NullDepthValue, job.NullTimeValue, logCurves);
+
+                        foreach (var mnemonicToDelete in mnemonicsToDelete)
+                        {
+                            var result = await DeleteMnemonic(wellboreRef.WellUid, wellboreRef.WellboreUid, logToCheck.Uid, mnemonicToDelete);
+
+                            if (result.IsSuccessful)
+                            {
+                                reportItems.Add(new DeleteEmptyMnemonicsReportItem { WellboreReference = wellboreRef, LogCurveInfo = mnemonicToDelete });
+
+                                Logger.LogInformation("Successfully deleted empty mnemonic. WellUid: {WellUid}, WellboreUid: {WellboreUid}, Uid: {LogUid}, Mnemonic: {Mnemonic}"
+                                    , wellboreRef.WellUid, wellboreRef.WellboreUid, logToCheck.Uid, mnemonicToDelete.Mnemonic);
+                            }
+                            else
+                            {
+                                Logger.LogWarning("Failed to delete empty mnemonic. WellUid: {WellUid}, WellboreUid: {WellboreUid}, Uid: {LogUid}, Mnemonic: {Mnemonic}"
+                                    , wellboreRef.WellUid, wellboreRef.WellboreUid, logToCheck.Uid, mnemonicToDelete.Mnemonic);
+                            }
+                        }
+                    }
+                }
+            }
+
+            var report = new DeleteEmptyMnemonicsReport() { ReportItems = reportItems };
+            job.JobInfo.Report = report;
```

Comment:
Fill out the report.Title and report.Summary fields. The summary should include the null values used, the total number of mnemonics that were checked, and, if no mnemonics were deleted, a message saying that.